bin/setup_new

```diff
@@ -21,17 +20,17 @@ init (){
 ## utility functions to test dependencies
 is_darwin (){
   PLATFORM=$(uname)
-  [ "$PLATFORM" == "Darwin" ]
+  [[ "$PLATFORM" == "Darwin" ]]
 }
 
 has_homebrew (){
-  [ is_darwin ] && [ hash brew >/dev/null 2>&1 ]
+  is_darwin && [ hash brew >/dev/null 2>&1 ]
 }
 
 verify_homebrew (){
-  if [ ! has_homebrew ]
```

Comment:
This wasn't working as intended with the square brackets, and I *think* it was because the `[` command was evaluating the exit code for `! has_homebrew` instead of the result of `! homebrew`, so it was always getting true even if homebrew wasn't installed.